pkg/apis/pipeline/v1/pipeline_types.go

```diff
@@ -238,6 +245,13 @@ type PipelineTask struct {
 	// Note: PipelineSpec is in preview mode and not yet supported
 	// +optional
 	PipelineSpec *PipelineSpec `json:"pipelineSpec,omitempty"`
+
+	// OnError defines the exiting behavior of a PipelineRun on error
+	// can be set to [ continue | stopAndFail ]
+	// Note: OnError is in preview mode and not yet supported
+	// TODO(@QuanZhang-William)
```

Comment:
nit: could you link this to an issue instead of yourself?

Reply:
Sure, I've created a tracking issue and updated the link